lib/logger.js

```diff
@@ -13,7 +13,7 @@ const transports = [
     prettyPrint: true,
     level: config.log_level || 'info',
     silent: false,
-    colorize: true,
+    colorize: (config.colorize_log !== undefined && config.colorize_log === false) ? false : true,
```

Comment:
by default this wont be defined. So only in cases where it is explicity defined and set to false, this is to be turned off. Existing behavior will not be affected by this code, this is only required in local and hence changes will be done only in brokers settings.yml and no changes required in bosh release.